docs/reference/hubs.md

```diff
@@ -1,16 +1,16 @@
+
 # List of running hubs
 
-Here's a list of running hubs.
+% NOTE: The CSV files used by this page are generated by `docs/scripts/render_hubs.py`
+
+Here's a list of running hubs and some statistics about them.
 It is automatically generated from the config stored in the [`config/clusters` folder of the `infrastructure` repository](https://github.com/2i2c-org/infrastructure/tree/HEAD/config/clusters) and can therefore be limited in the information it provides.
 
-```{admonition} Missing information for Azure hubs
```

Comment:
I wouldn't remove this. There's a reason we say "most likely running on Azure". Testing for `kubeconfig` is not a robust test for azure based clusters, even though it's true right now. We could export a `kubeconfig` for any cluster because it is a _kubernetes specific thing_, not a cloud vendor specific thing.

Reply:
is there a more reliable way to infer the cloud provider from the information in there? Maybe I am using the wrong field?

Reply:
The only reliable field is cluster provider, so I don't think we can reliably say "this is an Azure cluster" right now. This is because we only run one, and I doubt we have the correct permissions on that account to create the Azure version of service account and hence have the azure-specific config to authenticate. This is when we fall back onto `kubecobfig`

Reply:
What do you recommend we do here? Just list "kubeconfig" in the table instead of azure?

Reply:
Yeah, I think so. That was the decision I came to when I first introduced that column.

Reply:
ok will do in next push